src/zjs_timers.h

```diff
@@ -3,7 +3,13 @@
 #ifndef __zjs_timers_h__
 #define __zjs_timers_h__
 
-void zjs_timers_process_events();
+/*
```

Comment:
For JSDoc, you're supposed to have a second star on the first line: `/**`. Not that we're actually parsing any of this yet anyway, but just in preparation.